docs/pages/community.md

```diff
@@ -23,6 +23,10 @@ Our [GitHub discussions forum][git-hub-discussions-forum] is a good place to cha
 
 You can find (and create) examples in the [Remix examples repository][the-examples-repository].
 
+## Community Adapters
+
+You can find Community Adapters int the [Adapters Documentation][community-adapters]. If you've written your own, you can [update this list on GitHub][community-adapters-github].
```

Comment:
```suggestion
You can find Community Adapters in the [Adapters][community-adapters] section of the documentation. 

If you've written your own adapter, and think it will be valuable for others, you can [update this list on GitHub][community-adapters-github].
```